core/src/panic.rs

```diff
@@ -19,18 +15,17 @@ pub fn configure_panic() {
             Some(s) => *s,
             None => match panic_info.payload().downcast_ref::<String>() {
                 Some(s) => &s[..],
-                None => "unknown",
+                None => "Box<dyn Any>",
```

Comment:
interesting, what would it produce?

Reply:
or it's still the string?

Reply:
Just copying https://doc.rust-lang.org/src/std/panicking.rs.html#256

Reply:
i see, thanks :)

Reply:
We can also fund a way to use this https://doc.rust-lang.org/src/core/panic/location.rs.html#198 but I thing we're good